boot/service32.s

```diff
@@ -415,13 +415,16 @@ GDT64:  ; Global Descriptor Table (64-bit).
         dw $ - GDT64 - 1    ; Limit.
         dw GDT64, 0         ; 64 bit Base.
 
-        
+[BITS 64]
```

Comment:
For the record, this requires the assembler to support 64-bit code with elf32 output format, which is not the case with NASM version 2.11.08 and older. If you get "error: elf32 output format does not support 64-bit code", update NASM to a newer version (2.13.02 has been verified to work).

Reply:
Thanks, I will add this note to Build Envs in the wiki. nasm 2.11.08 appears to be over four years old, so I think we should be safe here.